### PR TITLE
Tickets/dm 33673

### PR DESCRIPTION
**Pull request by pcortes80**
Modified attributes related to queue´s size/timeout, and updated dependences/documentation.

### CODE REVIEW

File: simulator/README.md [hunk 1]
```diff
@@ -37,10 +37,10 @@ Please check:
 - telServerPort = 8889
 - serverTimeout (ms) = 100
 - serverBytesToRead = 32768
-- maxSizeQueueTel = 1000
+- queueTcpClientFromDdsMaxSize = 1000
```
Comment (teweitsai) [inline simulator/README.md:40]: maxSizeQueueTcpClientFromDds

File: simulator/README.md [hunk 2]
```diff
@@ -37,10 +37,10 @@ Please check:
...
 - EnqueueOptions = lossy enqueue
-- timeoutQueueTel (ms) = 50
-- tcpClientQueueMaxSize = 100
+- queueTcpClientFromDdsTimeout (ms) = 50
```
Comment (teweitsai) [inline simulator/README.md:42]: timeoutQueueTcpClientFromDds

File: simulator/README.md [hunk 3]
```diff
@@ -37,10 +37,10 @@ Please check:
...
+- queueTcpClientMaxSize = 100
```
Comment (teweitsai) [inline simulator/README.md:43]: maxSizeQueueTcpClient

**Review by teweitsai — CHANGES_REQUESTED** [review]
Please rename the attributes and accessor methods.

File: doc/versionHistory.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
 # Version History
 
+0.3.7
+
+- Rename **tcpClientQueueMaxSize** by **queueTcpClientMaxSize** and its dependences.
+- Rename **maxSizeQueueTel** by **queueTcpClientFromDdsMaxSize** and its dependences.
+- Rename **telQueueServerTimeout** and **timeoutQueueTel**  by **queueTcpClientFromDdsTimeout** and its dependences.
+- Modify **isQueueTcpClientFull.vi**, **getTelElement.vi** and **flushTelQueue.vi** in `TcpServer` class, so the user needs to update this vis in their callers."
```
[the remark this answers is not in the document]
Comment (pcortes80) [reply doc/versionHistory.md]: Done.

File: simulator/README.md (hunk 2 above)
Comment (pcortes80) [reply simulator/README.md]: Done.

File: simulator/README.md (hunk 3 above)
Comment (pcortes80) [reply simulator/README.md]: Done.

**Pull request closed without merging**